core/src/main/scala/kafka/raft/RaftManager.scala

```diff
@@ -104,6 +104,8 @@ trait RaftManager[T] {
   def replicatedLog: ReplicatedLog
 
   def voterNode(id: Int, listener: ListenerName): Option[Node]
+
+  def getRecordSerde: RecordSerde[T]
```

Comment:
In Kafka we generally don't put 'get' in front of getters. So this method should just be `recordSerde`